src/controller/playRace.js

```diff
@@ -1,33 +1,47 @@
 import { CAR_CAN_GO_COUNT } from '../util/constants.js';
+
 import generateCars from '../model/generateCars.js';
-import showResult from '../view/showResult.js';
 import getWinners from '../model/getWinners.js';
+
+import addGameProcessTemplate from '../view/addGameProcessTemplate.js';
 import showWinners from '../view/showWinners.js';
 import showRestart from '../view/showRestart.js';
+import { spinSpinner, hideSpinner } from '../view/viewSpinner.js';
+import alertWinners from '../view/alertWinners.js';
 
-function sortCars(cars) {
-  const sortedCars = [...cars];
-  return sortedCars.sort((a, b) => b.position - a.position);
+function isCarCanGo() {
+  return Math.floor(Math.random() * 10) >= CAR_CAN_GO_COUNT;
 }
 
-function isCarCanGo() {
-  if (Math.floor(Math.random() * 10) >= CAR_CAN_GO_COUNT) {
-    return true;
-  }
-  return false;
+function updateGameProcess() {
+  const cars = document.querySelectorAll('.result-car-wrapper');
+  cars.forEach(car => {
+    if (isCarCanGo()) {
+      const arrowContainer = car.querySelector('.result-arrow-container');
+      arrowContainer.insertAdjacentHTML('beforeend', '<div class="result-arrow">⬇️️</div>');
+    }
+  });
+}
+
+function endGame(cars) {
+  hideSpinner(document.querySelectorAll('.spinner'));
+  showWinners(getWinners(cars));
+  alertWinners(getWinners(cars));
```

Comment:
함수를 두 번 호출하지 않아도 좋을 거 같아요~
```suggestion
  const winners = getWinners(cars)
  showWinners(winners)
  alertWinners(winners)
```